src/styles/_article.scss

```diff
@@ -9,13 +9,21 @@
     h4,
     p {
       margin-top: 20px;
+      a {
+        color: black;
+        text-decoration: underline;
+      }
+      strong {
+        display: inline;
```

Comment:
`<strong>` is `display:inline` by default. Is there a reason you declare this? Are you maybe trying to overwrite something else in our code?